tokens/main.py

```diff
@@ -63,7 +68,10 @@ def main():
 
     config = {}
     for dataset, allowed_repos in ALLOWED_REPOS.items():
-        entries = {'projectId': get_project_id(dataset), 'allowedRepos': allowed_repos}
+        entries = {
+            'projectId': get_project_id(dataset),
+            'allowedRepos': ALWAYS_ALLOWED_REPOS + allowed_repos,
```

Comment:
To avoid dupes, not a big deal either way though.
```suggestion
            'allowedRepos': list(set(ALWAYS_ALLOWED_REPOS + allowed_repos)),
```

Reply:
Good catch, thanks!